Improve test suite (#81)
Mapotempo/optimizer-api: README.md

Run tests:
```
APP_ENV=test bundle exec rake test
```

> If you want to get information about how long last each test:

Reviewer: ```suggestion
If you want to get information about how long each test lasts:
```

```
TIME=true HTML=true APP_ENV=test bundle exec rake test
```

> This will generate a report with every test time. You can find this report in optimizer-api/test/html_reports folder.

Reviewer: ```suggestion
This generates a report with test times. You can find the report in optimizer-api/test/html_reports folder.
```

APP_ENV=test ROUTER_URL=http://localhost:4899 bundle exec rake test
```

If you want to get information about how long last each test:
```

> TIME=true HTML=true APP_ENV=test ROUTER_URL=http://localhost:4899 bundle exec rake test

Reviewer 2: We currently avoid the router in every tests ?

Author: Yes, I do not know why this is like that. Did we use to call router within tests ?

Reviewer 2: No, it makes the tests sensitive to the technology/version/config/data of the router

Author: So what was this ever used for ?

Author: In any case I should have removed this !

Reviewer 2: It was used before we manage to avoid the router calls in all the tests.
